OMCompiler/Compiler/BackEnd/SymbolicJacobian.mo

```diff
@@ -2021,7 +2021,7 @@ algorithm
         indepVars = createInDepVars(inDiffVars, false);
 
         if Flags.isSet(Flags.JAC_DUMP) then
-          print("Crete symbolic Jacobianis from:\n");
+          print("Create symbolic Jacobianis from:\n");
```

Comment:
This should read Jacobians and not Jacobianis I think

Reply:
At least I found 50%